### ISSUE TITLE
sw.js 404 when using publicPath

### ISSUE THREAD

**Issue by derevandal** [nuxt-community/pwa-module#7]
Hi, i'm using github pages to host and I got 404 error on sw.js.
```js
A bad HTTP response code (404) was received when fetching the script.
/sw.js Failed to load resource: net::ERR_INVALID_RESPONSE
(index):1 Uncaught (in promise) TypeError: Failed to register a ServiceWorker: A bad HTTP response code (404) was received when fetching the script.

```
Nuxt config:
```js
module.exports = {
  /*
  ** Build configuration
  */
  build: {
    publicPath: 'https://derevandal.github.io/idepead-subscription-frontend/',
  }
}
```

I believe this happens because the script is not injected into the page with publicPath.

Could anyone help me?
Thanks

<div align="right"><sub><em>This question is available on <a href="https://nuxtjs.cmty.io">Nuxt.js</a> community (<a href="https://nuxtjs.cmty.io/nuxt-community/pwa-module/issues/c8">#c8</a>)</em></sub></div>

**Comment by pi0** [comment]
Hi. Would you please check against new 2.0.0 release? If problem still exists, providing generated `.nuxt/sw.plugin|template.js` and `static/sw.js` files may help.

**Comment by james2doyle** [comment]
I fixed this by creating a symbolic link from the template to the main file:

```sh
ln -s $PWD/.nuxt/sw.template.js $PWD/.nuxt/sw.js
```

This way, the `sw.js` is always synced to whatever `sw.js` is

**Comment by manniL** [comment]
Should be resolved since #19
